packages/tables/src/styled/StyledGroupRow.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
+import math from 'polished/lib/math/math';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { StyledBaseRow, IStyledRowProps } from './StyledRow';
+import { StyledCell } from './StyledCell';
+import { getLineHeight } from './StyledTable';
+
+const COMPONENT_ID = 'tables.group_row';
+
+const sizeStyles = (props: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  const height = `${props.theme.space.base * 8}px`;
+  const lineHeight = getLineHeight(props);
+
+  return css`
+    height: ${height};
+    line-height: ${lineHeight};
```

Comment:
Out of curiosity, would we be able to use the newly added [`getLineHeight`](https://github.com/zendeskgarden/react-components/blob/next/packages/theming/src/utils/getLineHeight.ts#L18) utility here?

Reply:
This value is used for multiple styles. This allows the value to only be calculated once, instead of for `line-height` and the `padding` style below.

Reply:
Excellent, thanks for the clarification! I think these are some benefits of keeping the values separate, and supplying them to padding and line-height, respectively. 

1) The adoption of a standard a way to add line-height, via JZ's `getLineHeight` util. A developer would not have to choose between using the util function or calculating their own `line-height` with `space.base`. Also, line-heights can always be unitless in Garden.

2) `getLineHeight` is named the same as `getLineHeight` from `react-theming`. Not a big deal, but there is a chance that someone could confuse this with the util function from `react-theming` that is named the same. Particularly if they didn't follow through the code to identify where the import is coming from.

3) Another scenario might be that a new contributor sees `getLineHeight` in this file first, and think that this is the idiomatic way to set line-heights in Garden. But then in code review, they are told to use the `getLineHeight` util from `react-theming`, rather than calculating their own line-height via `space.base`.

Reply:
@hzhu this is a semi-unique styling where `line-height` needs to be expressed as pixels in order to workaround Chrome rounding bugs. We wouldn't want to use the generalized (unitless) `getLineHeight` utility here.

Reply:
Thanks, @jzempel, for the extra clarification. 

I ran this branch locally with the `getLineHeight` utility and applied a unitless `line-height` to `StyledGroupRow` to explore the Chrome rounding bugs. There appear to be no rounding issues, and the computed `line-height` seems to remain at `20px` for both the element and its descendants. Also, there seem to be no visual issues. I could be missing something, though, would you be able to confirm?